features/createroom/impl/src/test/kotlin/io/element/android/features/createroom/impl/configureroom/ConfigureRoomPresenterTests.kt

```diff
@@ -182,6 +185,10 @@ class ConfigureRoomPresenterTests {
             // From camera
             val uriFromCamera = Uri.parse(AN_URI_FROM_CAMERA)
             fakePickerProvider.givenResult(uriFromCamera)
+            assertThat(newState.cameraPermissionState.permissionGranted).isFalse()
```

Comment:
Shouldn't we also test that when permission is not granted we catually call into the PermissionsPresenter to request it?

Reply:
You're right, there is a missing test. I will ad it.